functions/src/submitVolunteerApplication/models.js

```diff
@@ -0,0 +1,77 @@
+const cleaner = require('deep-cleaner');
+
+exports.Volunteer = function Volunteer(data) {
+  const volunteer = {
+    // contains all info about a hacker
+    volunteer_full_info: {
+      id: data.id,
+      timestamp: data.timestamp,
+      // PAGE ONE
+      firstName: data.firstName,
+      lastName: data.lastName,
+      email: data.email,
+      city: data.city,
+      school: data.school,
+      gender: data.gender,
+      isOver19: data.isOver19,
+      education: data.education,
+      gradYear: data.gradYear,
+      major: data.major,
+      phoneNumber: data.phoneNumber,
+      ethnicity: data.ethnicity,
+      // PAGE TWO
+      interestForNwHacks: data.interestForNwHacks,
+      freeHours: data.freeHours,
+      day1_830_to_1: data.day1_830_to_1,
```

Comment:
The availability keys/values need to be updated in this file

Reply:
good catch, there are so many different files to change this in...
And it doesn't seem like my IDE can handle refactoring :(